src/pages/SkillsPage.tsx

```diff
@@ -0,0 +1,13 @@
+import { PageHeading } from '@/components/PageHeading';
+import { MarqueeSkills } from '@/components/MarqueeSkills';
+
+export function SkillsPage() {
+    return (
+        <div className="w-full h-full flex flex-col justify-center gap-8">
+            <PageHeading title="Skills" />
+
+            <MarqueeSkills direction="forward" />
+            <MarqueeSkills direction="reverse" />
```

Comment:
Nit: Once we provide the property optional and add a default value.

```suggestion
            <MarqueeSkills />
            <MarqueeSkills direction="reverse" />
```

Reply:
Made the `direction` prop optional with default value.
https://github.com/dhruvdhaduk-simform/portfolio/pull/2/commits/52dcc7402c7fbbea2f719dcfa82ad035e9c0a254